util/metrics/src/service_metrics.rs

```diff
@@ -64,31 +65,45 @@ pub struct ServiceMetrics {
     /// Histogram of message sizes for each gRPC message type tracked
     message_size: HistogramVec,
 }
+impl Default for ServiceMetrics {
+    fn default() -> Self {
+        let args = std::env::args().next().unwrap_or_default();
+        let mut arg = Path::new(&args).file_stem().unwrap_or_default().to_str().unwrap_or_default().replace('-', "_");
+        if !arg.is_empty() {
+            arg.push('_');
+        }
+        arg.push_str("grpc");
+        let svc = ServiceMetrics::new(arg);
```

Comment:
It looks like this call could be to `new_and_registered` instead of replicating the logic here.